platforms/hyperledger-fabric/charts/orderernode/templates/configmap.yaml

```diff
@@ -34,7 +34,7 @@ data:
   ORDERER_KAFKA_RETRY_SHORTTOTAL: "30s"
   ORDERER_KAFKA_VERBOSE: "true"
   GODEBUG: "netdns=go"
-  ORDERER_OPERATIONS_LISTENADDRESS: 0.0.0.0:9443
+  ORDERER_OPERATIONS_LISTENADDRESS: 0.0.0.0:10443
```

Comment:
It is necessary because if it is not modified the following error is produced in orderers pods
<img width="756" alt="image" src="https://github.com/hyperledger/bevel/assets/83813093/aa1b575f-382f-46fc-9f8c-d536f5c1ea74">